stable/artifactory/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
-version: 7.7.0
+version: 0.7.1
```

Comment:
@valdisrigdon Looks like you updated the plugin to `artifactory-ha` and bumped version of `artifactory` chart. We like to keep both our charts in sync. Can you add this changes in both charts?

Reply:
I looked at doing this, but since User Plugins looks like an Artifactory Pro feature, I didn't add it to the non-HA version.